spec/workers/fetch_graph_worker_spec.rb

```diff
@@ -29,7 +29,7 @@
       before do
         allow(controlled_val).to receive(:fetch).with(:anything).and_return({})
         allow_any_instance_of(OregonDigital::ControlledVocabularies::Creator).to receive(:solrize).and_return(['http://opaquenamespace.org/ns/creator/ChabreWayne', { label: 'Chabre, Wayne$http://opaquenamespace.org/ns/creator/ChabreWayne' }])
-        allow(worker).to receive(:fetch_failed_callback).with(nil, controlled_val).and_return(true)
+        # allow(worker).to receive(:fetch_failed_callback).with(nil, controlled_val).and_return(true)
```

Comment:
Same with this line. I think it can be removed if it's not needed

Reply:
These commented pieces of code will need to be reset. Its on my agenda for #986

Reply:
sounds good